Tasks/AzureFunctionAppContainerV1/azurefunctiononcontainerprovider.ts

```diff
@@ -48,8 +49,10 @@ export class AzureFunctionOnContainerDeploymentProvider{
 
         let containerDeploymentUtility: ContainerBasedDeploymentUtility = new ContainerBasedDeploymentUtility(this.appService);
         await containerDeploymentUtility.deployWebAppImage(this.taskParams);
-
-        this.taskParams.AppSettings = this.taskParams.AppSettings ? this.taskParams.AppSettings.trim() + " " + linuxFunctionStorageSetting : linuxFunctionStorageSetting;
+        const linuxFunctionStorageSetting: string = `${linuxFunctionStorageSettingName} ${linuxFunctionStorageSettingValue}`;
```

Comment:
suggestion: You can set value of `linuxFunctionStorageSetting` based on `this.taskParams.AppSettings.indexOf(linuxFunctionStorageSettingName) < 0`
```js
const linuxFunctionStorageSetting: string = '';
if (this.taskParams.AppSettings.indexOf(linuxFunctionStorageSettingName) < 0) {
    linuxFunctionStorageSetting = `${linuxFunctionStorageSettingName} ${linuxFunctionStorageSettingValue}`;
}
```

Reply:
@AmrutaKawade  As suggested Changed the code to set linuxFunctionStorageSetting based on condition